lib/roi.ts

```diff
@@ -83,6 +85,9 @@ export function calculateROI({
   const roundsCount = Math.round(
     (monthsForTimeHorizon * SECONDS_IN_A_MONTH) / averageSecondsPerRound
   );
+  
+  // Subtract the treasury reward cut rate from the inflation rate.
+  inflation = inflation * (1 - treasuryRewardCut);
```

Comment:
I think this should work, but I haven't yet found the time to research the exact calculations done using this method.